.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request, workflow_call, workflow_dispatch]
```

Comment:
Drop `workflow_call` from this given https://github.com/nextstrain/augur/pull/910 changes.

Reply:
Done.